src/merge_tools.rs

```diff
@@ -451,7 +453,15 @@ fn get_diff_editor_from_settings(
         CommandNameAndArgs::String(name) => get_tool_config(settings, name)?,
         CommandNameAndArgs::Vec(_) => None,
     };
-    Ok(maybe_editor.unwrap_or_else(|| MergeTool::with_edit_args(&args)))
+    let mut editor = maybe_editor.unwrap_or_else(|| MergeTool::with_edit_args(&args));
+    // If no interpolation parameters found, fall back to "$left $right"
+    // (TODO: or error out like merge-args?)
```

Comment:
This might be a bit controversial, but I'd keep defaulting to appending `$left $right` since it's convenient and correct in most cases.